packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDepositRow.tsx

```diff
@@ -261,110 +272,138 @@ export function TransactionsTableDepositRow({
     () =>
       sanitizeTokenSymbol(tx.asset, {
         erc20L1Address: tx.tokenAddress,
-        chain: l1.network
+        chain: getWagmiChain(tx.parentChainId)
       }),
-    [l1.network, tx.asset, tx.tokenAddress]
+    [tx.parentChainId, tx.asset, tx.tokenAddress]
   )
 
   const customAddressTxPadding = useMemo(
     () => (isCustomDestinationAddressTx(tx) ? 'pb-11' : ''),
     [tx]
   )
 
+  const rowHeight = useMemo(() => {
+    if (isPending(tx) || isFailed(tx)) {
+      return isCustomDestinationAddressTx(tx) ? 'h-[126px]' : 'h-[94px]'
+    }
+    return isCustomDestinationAddressTx(tx) ? 'h-[117px]' : 'h-[85px]'
```

Comment:
With the reskin I will create a single component for both withdrawals and deposits. I think we can keep it as is for now because it will be removed anyway